test/views/html/TaxFreeAmountViewSpec.scala

```diff
@@ -37,9 +37,9 @@ class TaxFreeAmountViewSpec extends TaiViewSpec {
         doc must haveDivWithId("taxFreeAmountSummary")
         doc must haveElementAtPathWithText(
           "div[id=taxFreeAmountSummary]>h2",
-          s"${messages("tai.taxFreeAmount.summarysection.heading")} £2020")
+          messages("tai.taxFreeAmount.summarysection.heading"))
```

Comment:
should "2020" have been removed here?

Reply:
yes intentionally removed  as the figure has moved outside of this element